src/app/(tabs)/event/page.tsx

```
@@ -56,27 +41,39 @@ export default function EventPage() {
+              router.replace(`?${params.toString()}`);
+            }}
             className={`relative px-3 pb-2 transition-colors ${
-              topTab === tab ? "text-white" : "text-gray-700"
+              tabParam === tab.value ? "text-white" : "text-gray-700"
             }`}
           >
-            {tab}
-            {topTab === tab && (
+            {tab.label}
+            {tabParam === tab.value && (
               <span className="absolute bottom-0 left-0 h-[2px] w-full rounded-full bg-white" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)